src/html_json.js

```diff
@@ -13,18 +13,24 @@ const moment = require('moment');
 const { JSDOM } = require('jsdom');
 const jsep = require('jsep');
 const { IndexConfig } = require('@adobe/helix-shared');
-const fetchAPI = require('@adobe/helix-fetch');
-
-const { fetch } = process.env.HELIX_FETCH_FORCE_HTTP1
-  ? fetchAPI.context({ alpnProtocols: [fetchAPI.ALPN_HTTP1_1] })
+const { fetch } = require('@adobe/helix-fetch').context({
+  httpsProtocols:
   /* istanbul ignore next */
-  : fetchAPI;
+    process.env.HELIX_FETCH_FORCE_HTTP1 ? ['http1'] : ['http2', 'http1'],
+});
```

Comment:
this project is already using helix-fetch v2:
https://github.com/adobe/helix-index-pipelines/blob/a884581c26cfd913e0f863334fb8d7ffa2076e16/package.json#L34

so keep the `alpnProtocols` way of  forcing http/1.1

Reply:
Somehow, this doesn't work: in the nock statements in `html_json.test.js`:
```
  before(async () => {
    nock('https://main--helix-index-pipelines--adobe.project-helix.page')
      .get((uri) => {
-->     return uri.startsWith('/test/specs/hlx_up');
      })
      .reply(200, (uri) => {
        const path = p.resolve(SPEC_ROOT, 'hlx_up', p.basename(uri).replace(/\.md$/, '.html'));
        return fse.readFile(path, 'utf-8');
      }, { 'last-modified': 'Mon, 22 Feb 2021 15:28:00 GMT' })
      .persist();
  });
```
the line marked `-->` never gets executed, although the hostnames *do* match

Reply:
OK, apparently passing `forceHttp1` as optional parameter to the service function **does** the trick, so I reverted the ALPN change, now tests are passing.

Reply:
see 16486da